pkgs/top-level/all-packages.nix

```diff
@@ -11763,7 +11763,7 @@ with pkgs;
 
   darwin = let
     apple-source-releases = callPackage ../os-specific/darwin/apple-source-releases { };
-  in apple-source-releases // rec {
+  in recurseIntoAttrs (apple-source-releases // rec {
```

Comment:
Well that's how it's always been actually. This PR didn't change any of that- just what is listed by Hydra changes.